versioning/scripts/dockerfiles/main.go

```diff
@@ -15,14 +15,55 @@ import (
 	"text/template"
 
 	"github.com/GoogleCloudPlatform/runtimes-common/versioning/versions"
+	"strconv"
```

Comment:
wrong order of imports. please ensure gofmt is run

Reply:
gofmt had no problems, manually moved into top imports